Library/Formula/contacts-phonetic.rb

```diff
@@ -0,0 +1,12 @@
+class ContactsPhonetic < Formula
+  desc "Add phonetic names to contacts."
+  homepage "https://github.com/Elethom/ContactsUtil-PhoneticName"
+  url "https://github.com/Elethom/ContactsUtil-PhoneticName/archive/0.1.1.tar.gz"
+  version "0.1.1"
```

Comment:
Homebrew determines Formula version from `url`. In this case, version will be `0.1.1`.
This lines is needless.